### PR TITLE
Tidy up from the hotfix for today's 500 error

### PR DESCRIPTION
**Pull request by alexwlchan**
Follows #7705

### CODE REVIEW

**Review by melanierogan — APPROVED** [review]
It looks good to me, with an annoying temporary caveat that I don't have things running locally on new laptop yet to properly check.

File: content/webapp/services/prismic/types/articles.ts [hunk 1]
```diff
@@ -40,7 +40,7 @@ export const eventsFetchLinks: FetchLinks<EventPrismicDocument> = [
   'events.schedule',
   'events.interpretations',
   'events.series',
-  'events.times',
+  'events.locations',
```
Comment (alexwlchan) [inline content/webapp/services/prismic/types/articles.ts:43]: That the Prismic TypeScript SDK will now catch this for us is super nice. 🎉

File: content/webapp/services/prismic/types/articles.ts [hunk 2]
```diff
@@ -40,7 +40,7 @@ export const eventsFetchLinks: FetchLinks<EventPrismicDocument> = [
...
   'events.place',
```
Comment (davidpmccormick) [inline content/webapp/services/prismic/types/articles.ts:44]: should this change be `place` instead of `times`?

Comment (alexwlchan) [reply content/webapp/services/prismic/types/articles.ts]: Well if you will be fussy and insist it's _correct_! :D

Comment (davidpmccormick) [reply content/webapp/services/prismic/types/articles.ts]: sorry to be soooooo finicky

**Review by davidpmccormick — APPROVED** [review]

**Comment by davidpmccormick** [comment]
Thanks for tidying up after me

**Pull request merged**